server/runtime_status.go

```diff
@@ -27,12 +28,31 @@ func (s *Server) Status(ctx context.Context, req *types.StatusRequest) (*types.S
 		networkCondition.Message = fmt.Sprintf("Network plugin returns error: %v", err)
 	}
 
-	return &types.StatusResponse{
+	resp := &types.StatusResponse{
 		Status: &types.RuntimeStatus{
 			Conditions: []*types.RuntimeCondition{
 				runtimeCondition,
 				networkCondition,
 			},
 		},
-	}, nil
+	}
+	if req.Verbose {
+		info, err := createRuntimeInfo(s)
+		if err != nil {
+			return nil, fmt.Errorf("creating runtime info: %w", err)
+		}
+		resp.Info = info
+	}
+	return resp, nil
+}
+
+func createRuntimeInfo(s *Server) (map[string]string, error) {
```

Comment:
I'd say we either pass the `s.config` or we make it a method of `*Server`.

Reply:
Maybe just pass the pause image, since it is an internal function anyway ?

Reply:
It is a bit of a mix:
```go
func (s *Server) createContainerInfo(container *oci.Container) (map[string]string, error)
func createImageInfo(result *pkgstorage.ImageResult) (map[string]string, error)
```

Reply:
```go
func (s *Server) createRuntimeInfo() (map[string]string, error)
```